makefile

```diff
@@ -5,3 +5,8 @@ test:
 
 linter:
 	golangci-lint -v run ./...
+
+generate:
+	go run ./cmd/initialismer/*.go -target="mapping" > ./initialism.go
```

Comment:
I'm unsure why you are not using `go generate` with a `//go:generate in the code

Reply:
Thanks for replying to every single comment I did.

This one has no reply yet

Reply:
In my opinion:

- go generate -- generate non repo files
- make generate -- generate repo files

I didn't think much about it 🤔

Reply:
I thought about it because I recently faced it

https://github.com/donatj/ghemoji/blob/46a16b3778b0f835f3167a9fffa809f9ebc05513/methods.go#L5